### PR TITLE
[eme] Simple EME playback and events

### PR DESCRIPTION
**Pull request by mwatson2**

### CODE REVIEW

**Comment by mwatson2** [comment]
This PR is dependent on #3355

File: encrypted-media/clearkey-mp4-playback-temporary-setMediaKeys-first.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                        audioType:          contentitem.audio.type,
+                        videoType:          contentitem.video.type,
+                        initDataType:       'keyids',
+                        initData:           getInitData(contentitem,'keyids'),
+                        duration:           2 };
+
+        runTest(config);
+    </script>
+  </body>
+</html>
```
Comment (jdsmith3000) [inline encrypted-media/clearkey-mp4-playback-temporary-setMediaKeys-first.html]: What's the purpose of clear key temporary playback events?  Isn't temporary playback implicit for Clear Key delivery?

Also, how much of your new tests draw from the playback Clear Key tests in the Google folder?  If we rename these, we'll need to do a test mapping exercise in the tracking document.  Do you have a plan for doing that?  Perhaps we should post the tracking document in this folder and have changes to it done per Pull Request?

**Comment by jdsmith3000** [comment]
My main question has to do with updating tracing with new tests.  The tests here seem reasonable to merge.  LGTM.

File: encrypted-media/clearkey-mp4-playback-temporary-setMediaKeys-first.html (hunk 1 above)
Comment (mwatson2) [inline encrypted-media/clearkey-mp4-playback-temporary-setMediaKeys-first.html]: temporary-playback-events is basic playback with a temporary session and validation of the event sequence.

I expect that we will delete the Google folder. The plan was to migrate each of those tests into a generic script that could handle clear key + drm, mp4 and webm.

**Pull request merged**